splight_cli/hub/server/exceptions.py

```diff
@@ -0,0 +1,50 @@
+from json import loads
+
+
+class SpecFormatError(Exception):
```

Comment:
To avoid repeating the same exceptions for Components and Servers create a file `exceptions.py` in the directory `splight_cli/hub/` and put there the same exceptions for `Components` and `Servers`

Some exceptions may require a little change to give more information, but at least we should use the same exceptions in both cases